### ISSUE TITLE
Can't easily drag mutiple node into property shared by multiple node

### ISSUE THREAD

**Issue by ajreckof** [godotengine/godot#90497]
### Tested versions

409c71bdfd8e0a08fd726225ebdce16bc60a4570

### System information

macOS 13.3.0 - Vulkan (Forward+) - integrated Apple M1 Pro - Apple M1 Pro (8 Threads)

### Issue description

Since #90378 you can now easily drag and drop a batch of node inside a property of a node. Unfortunately this doesn't work for putting it in properties in multiple nodes. This is a small usability issue which is not urgent especially since improving this will most likely be hard. Maybe here the good solution would be to be able to pin the Inspector.

### Steps to reproduce

N/A

### Minimal reproduction project (MRP)

N/A

**Comment by AThousandShips** [comment]
This sounds like a proposal, not a bug

**Comment by ajreckof** [comment]
Yeah you might be right I thought of it has a bug as the workaround would allow it if I remember correctly.
